### ISSUE TITLE
windows glfw3.x64 dll appears to be incorrect

### ISSUE THREAD

**Issue by christopherhesse** [FlorianRhiem/pyGLFW#37]
I downloaded `glfw-1.8.5-py2.py3-none-win32.zip` and `glfw-1.8.5-py2.py3-none-win_amd64.zip` from pypi and unzipped them:

```
md5 glfw-win*/glfw3.dll
MD5 (glfw-win32/glfw3.dll) = cc96a6eed5b83cc4521a16499ddf7187
MD5 (glfw-win_amd64/glfw3.dll) = cc96a6eed5b83cc4521a16499ddf7187
```

It looks like the packaging is messed up and that these are both the 32 bit dll, any ideas?

```
MD5 (glfw-3.3.bin.WIN32/lib-vc2010/glfw3.dll) = cc96a6eed5b83cc4521a16499ddf7187
```

**Comment by FlorianRhiem** [comment]
I suspect that this might be due to temporary files of running `bdist_wheel` being reused despite being passed different `plat_name` flag. Currently the script only deletes the files that are created on purpose. If that's not it I'll just have to isolate the build processes into differing CI jobs.

**Comment by FlorianRhiem** [comment]
Apparently the build directory is reused by `bdist_wheel` without checking correctly whether files had changed. By deleting it after building each wheel, this issue should be avoidable. These are the wheels created that way:
[dist.zip](https://github.com/FlorianRhiem/pyGLFW/files/3934875/dist.zip)

**Comment by FlorianRhiem** [comment]
This was fixed in release v1.8.6.

**Comment by christopherhesse** [comment]
Thanks for fixing this!
